apps/vr-tests/screener.local.config.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
   apiKey: 'ff2096ee-7c7d-4e80-824b-9114ff43549c',
   resolution: '1024x768',
   baseBranch: 'master',
-};
\ No newline at end of file
+  branch: 'localTest'
```

Comment:
Will this cause conflicts if 5 different devs run screener multiple times as they iterate within the same hour? I suppose there is only 1 instance of the "localTest" branch from screener's POV which does not differentiate between the 5 different devs...

Reply:
yeah, if we get to a point where it gets confusing, we can try to pull some local variable or something. without this change the branch just says (default) and our reviewers have been confused by seeing these tests and failures show up. This change doesn't make anything less clear for devs, but it certainly clears things up for reviewers.

Reply:
as to your first question, this change simply makes the branch show up as "localTest" instead of "(default)".